src/base/static/components/organisms/place-list.js

```diff
@@ -0,0 +1,236 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "react-emotion";
+import { translate } from "react-i18next";
+import { connect } from "react-redux";
+import { placesSelector } from "../../state/ducks/places";
+import PlaceListItem from "../molecules/place-list-item";
+import { Button } from "../atoms/buttons";
+import { HorizontalRule } from "../atoms/layout";
+import { TextInput } from "../atoms/input";
+
+// But if you only use a few react-virtualized components,
+// And you're concerned about increasing your application's bundle size,
+// You can directly import only the components you need, like so:
+import AutoSizer from "react-virtualized/dist/commonjs/AutoSizer";
+import List from "react-virtualized/dist/commonjs/List";
+import CellMeasurer from "react-virtualized/dist/commonjs/CellMeasurer/CellMeasurer";
+import CellMeasurerCache from "react-virtualized/dist/commonjs/CellMeasurer/CellMeasurerCache";
+// In wepack 4, we can do the following:
+// import { AutoSizer, List } from 'react-virtualized'
+
+const cache = new CellMeasurerCache({
+  defaultHeight: 160,
+  fixedWidth: true,
+});
+
+const ListViewContainer = styled("div")({
+  backgroundColor: "#fff",
+  width: "100%",
+  // HACK: We are horizonally centering all content, and clipping the
+  // height at 80%, to work around a layout issue where the scrollbars
+  // are getting clipped off the right edge. This is because the html
+  // body element is larger than 100% for some reason...
+  // We should be able to fix this when we port over base.hbs.
+  height: "80%",
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+});
+
+const ListViewContent = styled("div")({
+  margin: "24px",
+  height: "100%",
+  width: "100%",
+});
+
+const ListHeader = styled("div")({
+  marginTop: "24px",
+  marginBottom: "24px",
+  display: "flex",
+  justifyContent: "space-between",
+});
+
+const SearchContainer = styled("div")({});
+const SearchButton = styled(Button)({ marginLeft: "16px" });
+const SortButton = styled(Button)({ marginRight: "16px" });
```

Comment:
I'd argue for keeping the visual design of the original sort buttons on the list view, instead of the larger outlined buttons we have now. In our design language, that would correspond to `SmallText` (0.75rem) and no outline.

I think the current `SortButton`s have too much weight. They're bigger than items in the header bar, so it throws off some of the balance of the page, and we don't make use of the outlined buttons elsewhere so they feel out of place.